plugin.xml

```diff
@@ -200,4 +200,22 @@
 
 
     </platform>
+    
+    <platform name="browser">
+
+        <config-file target="config.xml" parent="/*">
+            <feature name="Mixpanel">
+                <param name="browser-package" value="MixpanelPlugin"/>
+            </feature>
+        </config-file>
+
+        <js-module src="src/browser/MixpanelProxy.js" name="MixpanelProxy">
```

Comment:
can you shortly explain what is this file for and how it cooperates with the mixpanel snippet?

I see that it does ``` module.exports = window.mixpanel; ```
but then so does the mixpanel snippet in line 2961 init fn:
```window[PRIMARY_INSTANCE_NAME] = mixpanel = instance; ```
PRIMARY_INSTANCE_NAME is 'mixpanel'